src/textfont.ts

```diff
@@ -290,7 +300,7 @@ export class TextFont {
     if (!metric) {
       return 0.65 * this.pointsToPixels;
     }
-    return (metric.advanceWidth / this.resolution) * this.pointsToPixels;
+    return ((metric.advanceWidth ?? 1000) / this.resolution) * this.pointsToPixels;
```

Comment:
This looks like a silent failure. What's happening here?

Reply:
Two possible explanations:
- Not all the `FontGlyphs` define `advanceWidth` so a default value is required. See `csymDiagonalArrangementSlash` in `Bravura`
- This is not a `FontGlyphs` and is a `TextFontMetric` there was a bug in the code mixing both types. Using any of the two, the test pass fine.

Reply:
Gotcha. I think a default is a good idea, but this is not a good place for it (it will be hard to find in the future.) Maybe put it in `FontGlyph`?

Reply:
@0xfe One question prior to the change, the right type for `metric` is `FontGlyphs` or `TextFontMetric`?

Reply:
@AaronDavidNewman do  you have thoughts here?

Reply:
Fixed ?? no longer necessary